src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked.
+ */
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Pressable} from 'react-native';
+
+const propTypes = {
+    onSecondaryInteraction: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+};
+
+class PressableWithSecondaryInteraction extends Component {
+    constructor(props) {
+        super(props);
+        this.pressableRef = React.createRef();
+    }
+
+    componentDidMount() {
+        this.pressableRef.current.addEventListener('contextmenu', (e) => {
```

Comment:
Whenever you add a listener in `componentDidMount()` you should clean it up by removing in `componentWillUnmount()`. Gonna guess there is no `onContextMenu` prop?

Reply:
Good point! Yep, no `onContextMenu` prop